source/core/replica-set-high-availability.txt

```diff
@@ -15,29 +19,242 @@ Replica Set High Availability
    :depth: 1
    :class: singlecol
 
-:term:`Replica sets <replica set>` provide high availability using
-automatic :term:`failover`. Failover allows a :term:`secondary` member
-to become :term:`primary` if the current primary becomes unavailable.
+:term:`Replica sets <replica set>` use elections to select the 
+:term:`primary` for the cluster. Replica set elections support
+high availability by allowing clusters to automatically recover 
+normal operations after a primary failover event. Applications can
+continue to interact with the cluster during elections, mitigating
+the risk of downtime due to critical errors such as network 
+disruptions and hardware failure.
+
+Replica sets can trigger an election in response to a variety of events,
+such as:
+
+- :method:`initiating a replica set <rs.initiate()>`,
+- performing replica set maintenance using methods such as :method:`rs.stepDown()` or :method:`rs.reconfig()`, and
+- losing connectivity to the primary for more than the configured :rsconf:`timeout <settings.electionTimeoutMillis>` (10 seconds by default).
+
+In the following diagram, the primary node unavailability
+exceeded the 
+:rsconf:`configured timeout <settings.electionTimeoutMillis>`.
+One of the remaining secondaries holds an election in response to
+elect itself as the new primary and ensure continued write availability.
+
+.. include:: /images/replica-set-trigger-election.rst
+
+During an election the replica set can continue to serve read queries 
+if such queries are configured to 
+:ref:`run on secondaries <replica-set-read-preference>`. The
+replica set cannot process write operations until the election completes
+successfully. 
+
+.. include:: /includes/fact-retryable-writes-election.rst
+
+Since MongoDB 3.2, elections typically complete within two seconds
+in clusters running with the default 
+:rsconf:`protocolVersion: 1 <protocolVersion>`. For elections
+triggered by primary failover, the total time in which
+a cluster may operate without a primary should not typically exceed
+12 seconds assuming the default settings for automatic failover.
+See :ref:`replication-auto-failover` for more information.
+
+.. include:: /includes/fact-election-latency.rst
+
+If a majority of replica set members are inaccessible or unavailable, 
+the remaining members of the replica set cannot hold elections.
```

Comment:
They can hold elections, they just can't win one.

Reply:
Done. I tweaked this paragraph a bit in general as well.